### ISSUE TITLE
Constructors use and nested properties

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vadbch/orika#96]
```
What steps will reproduce the problem?

Look at joined TestCase

What is the expected output? What do you see instead?


Please use labels and text to provide additional information.


```

Original issue reported on code.google.com by `elaat...@gmail.com` on 17 Apr 2013 at 9:44

Attachments:
- [testfororika.zip](https://storage.googleapis.com/google-code-attachments/orika/issue-96/comment-0/testfororika.zip)

**Comment by GoogleCodeExporter** [comment]
```
It appears that this test case is already fixed (didn't have to do anything)
```

Original comment by `matt.deb...@gmail.com` on 21 Jun 2013 at 3:41

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.4.3 release
```

Original comment by `matt.deb...@gmail.com` on 28 Jun 2013 at 8:22
- Changed state: **Fixed**
